src/components/MyClaimCreateView/MyClaimCreateView.tsx

```diff
@@ -30,7 +30,7 @@ type State = {
   partialClaim: sdk.IPartialClaim
   name: string
   isValid: boolean
-  cType?: sdk.CType
+  cType?: CTypeWithMetadata
```

Comment:
The state being an actual `sdk.CType` is only of use when we build the claim, so I changed it to `CTypeWithMetadata` work with the updated g`etClaimInputModel`, which now expects an `CTypeWithMetadata`.